docs/core/tutorials/top-level-templates.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Template changes in .NET 6"
+description: The console app template now uses top-level statements. Understand what's changed and how to use existing learning materials with the new program.
+ms.date: 07/13/2021
+---
+# New templates generate top-level statements
+
+Starting with the .NET 6 Preview 7 SDK, the console app template generates the following code:
+
+```csharp
+Console.WriteLine("Hello, World!");
+// See https://aka.ms/new-console-template for more information
+```
+
+The new output uses two recent C# features that simplify the code you need to write for a program.  Traditionally, the console template generated the following code:
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MyApp // Note: actual namespace depends on the project name.
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello World!");
+        }
+    }
+}
+```
+
+These two forms represent the same program. Both are valid with C# 10.0.  When you use the newer version, you need to write the body of the `Main` method. You don't need to write the other program elements by hand. You have two options you can follow with existing tutorials:
+
+- Use the new program style, adding new top-level statements as you add features
+- Convert the new program style to the older style, with a `Program` class and a `Main` method.
+
+## Use the new program style
+
+The two features that make the new program simpler are top-level statements and global usings.
```

Comment:
```suggestion
The features that make the new program simpler are top-level statements, global usings, and implicit usings.
```

Implicit usings and global usings are different features. You can turn off implicit usings without affecting global usings.